opts/gpus.go

```diff
@@ -20,7 +20,10 @@ func parseCount(s string) (int, error) {
 		return -1, nil
 	}
 	i, err := strconv.Atoi(s)
-	return i, errors.Wrap(err, "count must be an integer")
+	if err != nil {
+		return 0, errors.Wrap(err, "count must be an integer")
```

Comment:
line 20 we return a -1, should this also perhaps be -1?

Reply:
Oh! I was close doing that, but that `-1` is a special value for "all"; https://github.com/docker/cli/blob/eb48cad302f40aabf6086165d241c813003f95b6/opts/gpus.go#L18-L21


Returning `0` keeps the same behavior as before this PR, because `strconv.Atoi` returns `0` when failing to convert.

In practice, it shouldn't _REALLY_ matter what we return, as the caller returns an error if this fails, but it _does_ mutate the struct (which likely would get discarded, but something we should check if that's true); https://github.com/docker/cli/blob/eb48cad302f40aabf6086165d241c813003f95b6/opts/gpus.go#L59-L63